scaleway/helpers_k8s.go

```diff
@@ -198,9 +199,31 @@ func clusterAutoscalerConfigFlatten(cluster *k8s.Cluster) []map[string]interface
 	autoscalerConfig["balance_similar_node_groups"] = cluster.AutoscalerConfig.BalanceSimilarNodeGroups
 	autoscalerConfig["expendable_pods_priority_cutoff"] = cluster.AutoscalerConfig.ExpendablePodsPriorityCutoff
 
+	// need to convert a f32 to f64 without precision loss
+	thresholdF64, err := strconv.ParseFloat(fmt.Sprintf("%f", cluster.AutoscalerConfig.ScaleDownUtilizationThreshold), 64)
+	if err != nil {
+		// should never happen
+		return nil
+	}
+	autoscalerConfig["scale_down_utilization_threshold"] = thresholdF64
+	autoscalerConfig["max_graceful_termination_sec"] = cluster.AutoscalerConfig.MaxGracefulTerminationSec
+
 	return []map[string]interface{}{autoscalerConfig}
 }
 
+func clusterOpenIDConnectConfigFlatten(cluster *k8s.Cluster) []map[string]interface{} {
+	oidcConfig := map[string]interface{}{}
```

Comment:
Could it be renamed? I'm confused whether it is `openidClientConfig` or `openidClusterConfig`